### PR TITLE
basic layout of articles

### PR DESCRIPTION
**Pull request by sinnedchan**

### CODE REVIEW

**Review by HMelesi — APPROVED** [review]
Hi Dennis! Nice work getting the articles displayed. I would suggest adding a loading pattern though so that a message/spinner is displayed while the articles are being fetched.

File: src/components/ArticleCard.jsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+export default function ArticleCard({ article }) {
+  return (
+    <div>
```
Comment (xmenbteam) [inline src/components/ArticleCard.jsx]: This should be a <li>

File: src/components/ArticleList.jsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        setArticles(data.articles);
+        setIsLoading(false);
+      });
+  }, []);
+
+  if (isLoading) {
+    return <h2> Articles Loading </h2>;
+  }
+  return (
+    <div>
```
Comment (xmenbteam) [inline src/components/ArticleList.jsx]: This should be a <ul>

File: src/pages/Articles.jsx [hunk 3]
```diff
@@ -0,0 +1,3 @@
+export default function Articles() {
```
Comment (xmenbteam) [inline src/pages/Articles.jsx]: Is this being used anywhere? I think this should be deleted so it doesn't confuse you with ArticleList.jsx

**Review by xmenbteam — CHANGES_REQUESTED** [review]
Great work! Just a few bits to change before this can be approved.

**Review by popatre — APPROVED** [review]
Looks good to me

**Pull request merged**